test/integration/use_int_test.go

```diff
@@ -126,6 +127,10 @@ func (suite *UseIntegrationTestSuite) TestReset() {
 	cp.Expect("Checked out project")
 	cp.ExpectExitCode(0)
 
+	zsh := &zsh.SubShell{}
```

Comment:
Technically this will be created on Windows. Do we care? Perhaps we do.
Please also consider adding a comment annotating why you're creating this file and what you intend to do with it. It's a bit jarring to see a zshrc created out of nowhere for a reset test.